src/runtime_src/core/pcie/driver/linux/xocl/lib/libxdma.c

```diff
@@ -30,6 +30,8 @@
 
 extern unsigned int desc_blen_max;
 
+#define xocl_pr_info(fmt, args...)				\
+	printk(KERN_DEBUG pr_fmt(fmt), ##args)
```

Comment:
can we use pr_debug() which preserves the msg format and also do KERN_DEBUG?

Reply:
pr_debug() will equivalent to printk only when DEBUG is defined.